tutorial/source/gp.ipynb

```diff
@@ -658,7 +658,7 @@
d7dZ+qY4x00YxaSaKibXVDGpppq60RXUVJczrrqcmupyakaVM7ayTJ9qFhmhDhsE7n7RofrN7HrgcmCOD5xw6AJaMoY1A+vC8lDtEoGJYyr52KlT+NipUwDY15ti5aZddG7YSeeGnXRs2MnaLbt5Zvkm1u/YS98QVx8lLP012aMqyqiuSFJdnqSqPEFVeZKq8vR6dXmSyvIE5ckEZUmjIpmgLJGgvKx/2SgvS/eXJy09LpGgomxguTxpJBPpn4QZZUkjaUYiYZSFtmT/csZj0gae1z9eRNJymhoys8uALwIXuHvmndUXAL80s2+SPlncCjxH+kih1cymAWuBa4D/nEsNkl8VZQlmNo5lZuPYg/r6Us7GnXvZtHMf297tYdu7PWwPj/0/u/f1saenj3d7+nh3Xx879/bSvWMve3r62NOTYk9vH719zr6+FD19KaL6Fm0zDgiQA8JkcHBkBIgZJMxIJNKPZkaif1uH6E+ER7P+1yL0DfRbxrj+5yYT/W3Z+w/admJgechtJ2ygjsH9GXUbmdvmwLYEGLa/ff92COPsEM+1/uelt7H/NbM9l4GxWZ+bUf8RPZeBemRArucIvgdUAo+EHfuMu/+Vu79uZveSPgncC3zG3fsAzOyzwMNAErjD3V/PsQYZJsmE0TiuisZxVXnbZl/K6Qmh0NN34HJvXyoExoHLqZTTl3L6PDymnJQ7vX3ptlTK6Q1t/f37xw8ac8D4QdvsS0FfKkWfE8akgyvl4J7efsoJj04qxf7X7Okb6PeMcX0pD9tI9w8sM2g9/fqDXyezP3PbQx2pydAOCpHQbpkhQkZfGJAZWJn9A9mS2XfgtmAghDIDbXB//+sZMHtKDd+99vRC7oqcrxqacYi+LwNfztL+IPBgLq8rpSP923Z6+khy44OC6aCQSWWEzOD+1OCQSa877A85J7RlvkZ43f6A3P+80JcZWOntDQRm1uce0JbludleIxXGZXsuA2MPVV+q/9B00HbSy+ntEJb793VmX8bTw/LAvsrW7+GFPGNb7O/LaHM4bkJ1gf7FDNCXzomUiPR0DySzXqshMjRdOC4iEnMKAhGRmFMQiIjEnIJARCTmFAQiIjGnIBARiTkFgYhIzCkIRERiLqcb0wwXM+sGVuWwiTpgY57KKYRirw+Kv8Zirw9UYz4Ue31QXDW+x93rDzdoRARBrsys/Uju0hOVYq8Pir/GYq8PVGM+FHt9MDJqHExTQyIiMacgEBGJubgEwfyoCziMYq8Pir/GYq8PVGM+FHt9MDJqPEAszhGIiMjQ4nJEICIiQyjpIDCzy8xsqZl1mtnNEdbRYmaPmdkSM3vdzD4X2ieY2SNm1hEex4d2M7PvhLpfMbMzhqnOpJm9aGa/DevTzOzZUN+vzKwitFeG9c7QP3WY6qs1s/vM7M2wL88upn1oZn8b/n5fM7O7zawq6n1oZneY2QYzey2j7aj3mZldH8Z3WPo+5YWu8evh7/kVM/uNmdVm9N0SalxqZpdmtBfk/Z6tvoy+vzczN7O6sB7JPsxZ+k48pfdD+laYy4HpQAXwMjA7olomA2eE5bHAMmA28C/AzaH9ZuBrYfkjwEOk71R3FvDsMNX5BeCXwG/D+r3ANWH5R8Cnw/JNwI/C8jXAr4apvjuB/x6WK4DaYtmHQBPwFlCdse/+W9T7EDgfOAN4LaPtqPYZMAFYER7Hh+XxBa7xEqAsLH8to8bZ4b1cCUwL7/FkId/v2eoL7S2kb7u7CqiLch/m/GeMuoCC/cHgbODhjPVbgFuirivU8gBwMbAUmBzaJgNLw/KPgWszxu8fV8CamoFFwIXAb8M/5I0Zb8b9+zP84z87LJeFcVbg+saF/2htUHtR7EPSQbAmvNHLwj68tBj2ITB10H+yR7XPgGuBH2e0HzCuEDUO6vs4cFdYPuB93L8fC/1+z1YfcB9wKrCSgSCIbB/m8lPKU0P9b8x+XaEtUmEK4HTgWaDR3d8GCI8NYVgUtd8G/AOQCusTga3u3pulhv31hf5tYXwhTQe6gZ+F6aufmtloimQfuvta4BvAauBt0vtkMcW1D/sd7T6L+r30KdK/ZXOIWoa1RjO7Aljr7i8P6iqK+o5WKQdBthu3RnqJlJmNAX4NfN7dtx9qaJa2gtVuZpcDG9x98RHWEMW+LSN9eP5Ddz8d2EV6WmMow70PxwPzSE9XTAFGA3MPUUPR/ftk6Joiq9XMbgV6gbv6m4aoZdhqNLNRwK3A/8zWPUQdxfj3vV8pB0EX6Tm8fs3AuohqwczKSYfAXe5+f2heb2aTQ/9kYENoH+7azwWuMLOVwD2kp4duA2rNrCxLDfvrC/01wOYC1tf/ml3u/mxYv490MBTLPrwIeMvdu929B7gfOIfi2of9jnafRfJeCidULwc+6WE+pUhqPJ504L8c3jPNwAtmNqlI6jtqpRwEzwOt4aqNCtIn5BZEUYiZGXA7sMTdv5nRtQDov3rgetLnDvrbrwtXIJwFbOs/lC8Ed7/F3ZvdfSrp/fSou38SeAy4aoj6+uu+Kowv6G837v4OsMbMTghNc4A3KJLMw0clAAABKklEQVR9SHpK6CwzGxX+vvvrK5p9mOFo99nDwCVmNj4c+VwS2grGzC4Dvghc4e67B9V+TbjqahrQCjzHML7f3f1Vd29w96nhPdNF+mKQdyiifXhUoj5JUcgf0mfwl5G+muDWCOv4IOnDwFeAl8LPR0jPCS8COsLjhDDegO+Hul8F2oax1g8xcNXQdNJvsk7g34HK0F4V1jtD//Rhqu00oD3sx/8gffVF0exD4H8BbwKvAf9G+sqWSPchcDfpcxY9pP/DuuFY9hnpefrO8PMXw1BjJ+k59f73y48yxt8aalwKzM1oL8j7PVt9g/pXMnCyOJJ9mOuPPlksIhJzpTw1JCIiR0BBICIScwoCEZGYUxCIiMScgkBEJOYUBCIiMacgEBGJOQWBiEjM/X/U3q9iAnV+wwAAAABJRU5ErkJggg==\n",
```

Comment:
The current plot is at http://pyro.ai/examples/gp.html#More-Sparse-GPs. Because of the bug, two standard deviations cover all points, which should indicate that something is wrong.